php/values.yaml

```diff
@@ -452,9 +452,1875 @@ fpm:
+      ; to proxy requests or to process the POST data in a memory efficient fashion.
+      ; http://php.net/enable-post-data-reading
+      ;enable_post_data_reading = Off
+
+      ; Maximum size of POST data that PHP will accept.
+      ; Its value may be 0 to disable the limit. It is ignored if POST data reading
+      ; is disabled through enable_post_data_reading.
+      ; http://php.net/post-max-size
+      post_max_size = 8M
+
+      ; Automatically add files before PHP document.
+      ; http://php.net/auto-prepend-file
+      auto_prepend_file =
+
+      ; Automatically add files after PHP document.
+      ; http://php.net/auto-append-file
+      auto_append_file =
+
+      ; By default, PHP will output a media type using the Content-Type header. To
+      ; disable this, simply set it to be empty.
+      ;
+      ; PHP's built-in default media type is set to text/html.
+      ; http://php.net/default-mimetype
+      default_mimetype = "text/html"
+
+      ; PHP's default character set is set to UTF-8.
+      ; http://php.net/default-charset
+      default_charset = "UTF-8"
+
+      ; PHP internal character encoding is set to empty.
+      ; If empty, default_charset is used.
+      ; http://php.net/internal-encoding
+      ;internal_encoding =
+
+      ; PHP input character encoding is set to empty.
+      ; If empty, default_charset is used.
+      ; http://php.net/input-encoding
+      ;input_encoding =
+
+      ; PHP output character encoding is set to empty.
+      ; If empty, default_charset is used.
+      ; See also output_buffer.
+      ; http://php.net/output-encoding
+      ;output_encoding =
+
+      ;;;;;;;;;;;;;;;;;;;;;;;;;
+      ; Paths and Directories ;
+      ;;;;;;;;;;;;;;;;;;;;;;;;;
+
+      ; UNIX: "/path1:/path2"
+      ;include_path = ".:/php/includes"
+      ;
+      ; Windows: "\path1;\path2"
+      ;include_path = ".;c:\php\includes"
+      ;
+      ; PHP's default setting for include_path is ".;/path/to/php/pear"
+      ; http://php.net/include-path
+
+      ; The root of the PHP pages, used only if nonempty.
+      ; if PHP was not compiled with FORCE_REDIRECT, you SHOULD set doc_root
+      ; if you are running php as a CGI under any web server (other than IIS)
+      ; see documentation for security issues.  The alternate is to use the
+      ; cgi.force_redirect configuration below
+      ; http://php.net/doc-root
+      doc_root =
+
+      ; The directory under which PHP opens the script using /~username used only
+      ; if nonempty.
+      ; http://php.net/user-dir
+      user_dir =
+
+      ; Directory in which the loadable extensions (modules) reside.
+      ; http://php.net/extension-dir
+      ; extension_dir = "./"
+      ; On windows:
+      ; extension_dir = "ext"
+
+      ; Directory where the temporary files should be placed.
+      ; Defaults to the system default (see sys_get_temp_dir)
+      sys_temp_dir = "/tmp"
```

Comment:
Specify the temporary directory because an error occurs when using a temporary file in the php stream.